.env

```diff
@@ -23,27 +23,7 @@ _APP_DB_SCHEMA=appwrite
 _APP_DB_USER=user
 _APP_DB_PASS=password
 _APP_DB_ROOT_PASS=rootsecretpassword
-_APP_STORAGE_DEVICE=Local
-_APP_STORAGE_S3_ACCESS_KEY=
-_APP_STORAGE_S3_SECRET=
-_APP_STORAGE_S3_REGION=us-east-1
-_APP_STORAGE_S3_BUCKET=
-_APP_STORAGE_DO_SPACES_ACCESS_KEY=
-_APP_STORAGE_DO_SPACES_SECRET=
-_APP_STORAGE_DO_SPACES_REGION=us-east-1
-_APP_STORAGE_DO_SPACES_BUCKET=
-_APP_STORAGE_BACKBLAZE_ACCESS_KEY=
-_APP_STORAGE_BACKBLAZE_SECRET=
-_APP_STORAGE_BACKBLAZE_REGION=us-west-004
-_APP_STORAGE_BACKBLAZE_BUCKET=
-_APP_STORAGE_LINODE_ACCESS_KEY=
-_APP_STORAGE_LINODE_SECRET=
-_APP_STORAGE_LINODE_REGION=eu-central-1
-_APP_STORAGE_LINODE_BUCKET=
-_APP_STORAGE_WASABI_ACCESS_KEY=
-_APP_STORAGE_WASABI_SECRET=
-_APP_STORAGE_WASABI_REGION=eu-central-1
-_APP_STORAGE_WASABI_BUCKET=
+_APP_STORAGE_CONNECTION=
```

Comment:
Should we keep or remove the `_APP_STORAGE_DEVICE ` environment variable ? 

The DSN strings have the device in them `s3://access_key:access_secret@host:port/bucket?region=us-east-1`

Reply:
Remove all the old vars. The name should be: `_APP_CONNECTIONS_STORAGE`